packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -133,6 +139,5 @@ module.exports = {
 		'jsdoc/require-returns-check': 'error',
 		'jsdoc/require-returns-description': 'error',
 		'jsdoc/require-returns-type': 'error',
-		'jsdoc/valid-types': 'error',
```

Comment:
So it looks like we use this rule for all non-TypeScript files:

https://github.com/WordPress/gutenberg/blob/7901895ca20cf61e402925e31571d659dab64721/.eslintrc.js#L241-L245

How many new errors does it report if we leave this rule enabled?

Reply:
It wasn't too much. I fixed some, and I just disabled others here: https://github.com/WordPress/gutenberg/pull/34338/commits/ffec4db403daf875d18ca89a56fef38673b1f138

The ones I disabled were using `{0|-1|null}` as type. As far as I found, we can have `{(1|2)}` (not negative) and `{('a'|'b')}`, but I didn't find how to combine a mix.

Reply:
Knowing how that works, it probably could work as a type defined separately and used in those places. Anyway, it eventually become non-issue when we migrate to TypeScript.